server/src-lib/Hasura/GraphQL/Transport/HTTP.hs

```diff
@@ -191,26 +220,25 @@ runQueryDB
      , MonadReader E.ExecutionCtx m
      , MonadQueryLog m
      , MonadTrace m
-     , MonadExecuteQuery m
      )
   => RequestId
-  -> (GQLReqUnparsed, GQLReqParsed)
-  -> [QueryRootField UnpreparedValue]
+  -> GQLReqUnparsed
   -> UserInfo
   -> (Tracing.TraceT (LazyTx QErr) EncJSON, EQ.GeneratedSqlMap)
-  -> m (DiffTime, Telem.QueryType, HTTP.ResponseHeaders, EncJSON)
+  -> m (DiffTime, Telem.QueryType, EncJSON)
   -- ^ Also return 'Mutation' when the operation was a mutation, and the time
   -- spent in the PG query; for telemetry.
-runQueryDB reqId (query, queryParsed) asts _userInfo (tx, genSql) =  do
+runQueryDB reqId query _userInfo (tx, genSql) =  do
   -- log the generated SQL and the graphql query
   E.ExecutionCtx logger _ pgExecCtx _ _ _ _ <- ask
   logQueryLog logger query (Just genSql) reqId
-  (telemTimeIO, respE) <- withElapsedTime $ runExceptT $ trace "Query" $
-    Tracing.interpTraceT id $ executeQuery queryParsed asts (Just genSql) pgExecCtx tx
-  (respHdrs,resp) <- liftEither respE
+  (telemTimeIO, respE) <- withElapsedTime $ runExceptT $ trace "Postgres Query" $
```

Comment:
Looks like we are changing the paths in the traces? `Query -> Postgres Query`, is that okay?

Reply:
This was discussed with @paf31; since this trace used to cover all that was done in `MonadExecuteQuery`, the top-level "Query" trace call has been moved to the calling function; this is now just the span where we call Postgres.